engine/src/main/java/io/camunda/zeebe/engine/processing/processinstance/CreateProcessInstanceProcessor.java

```diff
@@ -146,6 +146,15 @@ private boolean hasValidStartInstructions(
       final ExecutableProcess process,
       final ArrayProperty<ProcessInstanceCreationStartInstruction> startInstructions) {
 
+    return !hasStartInstructionsWithNonExistingElements(controller, process, startInstructions)
+        && !hasStartInstructionsWithMultiInstanceElement(controller, process, startInstructions);
+  }
+
+  private boolean hasStartInstructionsWithNonExistingElements(
```

Comment:
:wrench: Maybe return an `Either<String, boolean>` and reject the command in the caller method (same for other method)